packages/plugin-graphql/src/schema/schema.js

```diff
@@ -1,50 +1,78 @@
 const { makeExecutableSchema } = require('apollo-server-express');
 const { configTypeDefs, configResolvers } = require('./config');
 const { graphTypeDefs, graphResolvers } = require('./graph');
+const fs = require('fs');
 const gql = require('graphql-tag');
+const path = require('path');
 
-module.exports = (graph) => {
-  let uniqueCustomDataDefKeys = {};
-  let customDataDefs = '';
+module.exports = (compilation) => {
+  const { graph } = compilation;
+  const uniqueCustomDataDefKeys = {};
+  const customSchemasPath = `${compilation.context.userWorkspace}/data/schema`;
```

Comment:
So this is great for unique queries for each individual user.  If you want to extend it to have plugins include schema, we would need to do an additional directory walk of any plugins containing a graphql flag of somekind.